src/plugins/python/python.cpp

```diff
@@ -101,8 +101,9 @@ static int Python_CallFunction_Int (moduleData * data, PyObject * object, PyObje
 	PyObject * res = Python_CallFunction (object, args);
 	if (!res)
 	{
-		ELEKTRA_SET_ERRORF (111, errorKey, "Error while calling python function of script %s%s", keyString (data->script),
-				    data->printError ? "" : ", use /print to print error messages");
+		// TODO: Correct??
+		ELEKTRA_SET_INSTALLATION_ERRORF (errorKey, "Error while calling python function of script %s%s", keyString (data->script),
```

Comment:
So this error basically says that someone called the python plugin with parameters that led to a failure?

Reply:
I am not sure in which situations this can be triggered (there might be many) but it should be not so difficult to trigger such errors. You really should also try to trigger errors and not only think about it.

Note that src/plugins/python/testmod_python.c already triggers one error.

Reply:
After investigating the test who triggers the error it is clear that it is a plugin misbehavior because you do not get any result in all which indicates a broken plugin